working/augmentation-libraries/feature-specification.md

```diff
@@ -525,6 +464,11 @@ augment class C {
 }
 ```
 
+An extension type declaration induces an implicit unnamed constructor that is
```

Comment:
Not necessarily unnamed. `extension type const Foo._(int _) {}`

Reply:
Oh, good catch. Also, I just realized this is completely inconsistent with the section on augmenting extension types which says that you *can* augment the implicit constructor. I moved this paragraph down to that section and removed the inconsistent stuff.